### PR TITLE
Feat/transaction pending

### PR DESCRIPTION
**Pull request by simeonpp**

### CODE REVIEW

File: api/sql/schema/360$transfer.pending.sql [hunk 1]
```diff
@@ -1,12 +1,20 @@
 CREATE TABLE [transfer].[pending](
-	[pendingId] [int] IDENTITY(1000,1) NOT NULL,
-	[firstTransferId] [bigint] NULL,
-	[secondTransferId] [bigint] NULL,
-	[securityCode] [varchar](50) NULL,
-	[expireTime] [datetime] NULL,
-	[attempts] [int] NULL,
-	[status] [int] NULL,
+	[pendingId] [int] IDENTITY(1000,1) NOT NULL, -- id of the pending record
```
Comment (kalinkrustev) [inline api/sql/schema/360$transfer.pending.sql]: remove tabs

File: api/sql/schema/750$transfer.pendingTransfer.cancel.sql [hunk 2]
```diff
@@ -0,0 +1,43 @@
+ALTER PROCEDURE [transfer].[pendingTransfer.cancel] -- this SP is used to cancel opened pending transaction
```
Comment (kalinkrustev) [inline api/sql/schema/750$transfer.pendingTransfer.cancel.sql]: rename pendingTransfer => pending

File: api/sql/schema/750$transfer.pendingTransfer.reject.sql [hunk 3]
```diff
@@ -0,0 +1,44 @@
+ALTER PROCEDURE [transfer].[pendingTransfer.reject] -- this SP fetches the open pull requests of the user
```
Comment (kalinkrustev) [inline api/sql/schema/750$transfer.pendingTransfer.reject.sql]: rename pendingTransfer => pending

File: api/sql/schema/750$transfer.push.create.sql [hunk 4]
```diff
@@ -211,7 +268,18 @@ BEGIN TRY
...
+    IF @@TRANCOUNT > 0
+        ROLLBACK TRANSACTION
+
+    IF error_number() not in (2627)
+        BEGIN
+            EXEC [core].[error]
+        END
+    ELSE
+    BEGIN TRY
+        RAISERROR('Transfer ID already exists', 16, 1);
```
Comment (kalinkrustev) [inline api/sql/schema/750$transfer.push.create.sql]: this should raise error that exists in errors.js

File: api/sql/schema/750$transfer.pendingTransfer.cancel.sql (hunk 2 above)
Comment (simeonpp) [inline api/sql/schema/750$transfer.pendingTransfer.cancel.sql]: Done.

File: api/sql/schema/750$transfer.pendingTransfer.reject.sql (hunk 3 above)
Comment (simeonpp) [inline api/sql/schema/750$transfer.pendingTransfer.reject.sql]: Done.

File: api/sql/schema/750$transfer.push.create.sql (hunk 4 above)
Comment (simeonpp) [inline api/sql/schema/750$transfer.push.create.sql]: Done.

File: api/sql/schema/360$transfer.pending.sql (hunk 1 above)
Comment (simeonpp) [inline api/sql/schema/360$transfer.pending.sql]: Done.

**Pull request merged**